packages/@aws-cdk-testing/framework-integ/test/custom-resources/test/aws-custom-resource/integ.cross-account-assumeRole.ts

```diff
@@ -0,0 +1,113 @@
+import * as iam from 'aws-cdk-lib/aws-iam';
+import * as cdk from 'aws-cdk-lib';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import { AwsCustomResource, AwsCustomResourcePolicy, PhysicalResourceId } from 'aws-cdk-lib/custom-resources';
+
+/**
+ * Notes on how to run this integ test
+ * (All regions are flexible, my testing used account A with af-south-1 not enabled)
+ *
+ * 1. Configure Accounts
+ *   a. Account A (123456789) should be bootstrapped for us-east-1
+ *      and needs to set trust permissions for account B (987654321)
+ *      - `cdk bootstrap --trust 987654321 --cloudformation-execution-policies 'arn:aws:iam::aws:policy/AdministratorAccess'`
+ *      - assuming this is the default profile for aws credentials
+ *   b. Account B (987654321) should be bootstrapped for us-east-1 and af-south-1
+ *     - note Account B needs to have af-south-1 enabled as it is an opt-in region
+ *     - assuming this account is configured with the profile 'cross-account' for aws credentials
+ *
+ * 2. Set environment variables
+ *   a. `export CDK_INTEG_ACCOUNT=123456789`
+ *   b. `export CDK_INTEG_CROSS_ACCOUNT=987654321`
```

Comment:
```suggestion
 *   a. `export CDK_INTEG_ACCOUNT=123456789012`
 *   b. `export CDK_INTEG_CROSS_ACCOUNT=234567890123`
```

https://github.com/aws/aws-cdk/blob/main/.gitallowed